st2common/st2common/models/db/action.py

```diff
@@ -215,6 +209,9 @@ class ActionAliasDB(stormbase.StormBaseDB):
     formats = me.ListField(
         field=me.StringField(),
         help_text='Possible parameter formats that an alias supports.')
+    file_uri = me.StringField(
```

Comment:
Ah, that's actually an action alias and not action model. 

On a related note, `RuleDB` also doesn't have a pack attribute, but we still inherit from `StormContentDB` it looks like :)